main/fastcgi.c

```diff
@@ -747,7 +743,13 @@ int fcgi_listen(const char *path, int backlog)
 
 	/* Create, bind socket and start listen on it */
 	if ((listen_socket = socket(sa.sa.sa_family, SOCK_STREAM, 0)) < 0 ||
-#ifdef SO_REUSEADDR
+#if !defined(_WIN32) && defined(SO_REUSEADDR)
+       /* SO_REUSEADDR on windows has a different behavior compared to unixes.
+        * It potentially allow a same address/port combination, even from an active connection (aka port hijacking).
+        * We trade the "fast port reuse on failure" ability though but performance ought to take a backseat on this matter.
+        * It might be tempting to use SO_EXCLUSIVEADDRINUSE, but it can significantly increase the amount of binding failures
+        * without real benefits .. but it is open for discussions.
```

Comment:
note that even [rust forbids its usage on windows as well](https://github.com/rust-lang/rust/blob/cc63a0abde59a0de58ced46a2a882364f967632c/library/std/src/sys/net/connection/socket/mod.rs#L553).